cluster-autoscaler/simulator/dynamicresources/utils/utilization.go

```diff
@@ -110,6 +110,11 @@ func groupAllocatedDevices(claims []*resourceapi.ResourceClaim) (map[string]map[
 		}
 
 		for _, deviceAlloc := range alloc.Devices.Results {
+			if deviceAlloc.AdminAccess != nil && *deviceAlloc.AdminAccess {
```

Comment:
can shorten to `if ptr.Deref(devAlloc.AdminAccess, false) {` (have to import `"k8s.io/utils/ptr"` into this file as well)